messages/dji_msgs/msg/Topics.msg

```diff
@@ -17,6 +17,8 @@ string GIMBAL_CAMERA_RAW_TOPIC = 'gimbal_camera/image_raw' # Image
 string ESTIMATED_BUOY_TOPIC = 'alars_detection/buoy' # PointStamped
 string ESTIMATED_AUV_TOPIC = 'alars_detection/auv' # PointStamped
 string ESTIMATED_MIDDLE_TOPIC = 'alars_detection/middle' # PointStamped
+string ESTIMATED_CNN_TOPIC = 'alars_detection/cnn' # PointStamped
```

Comment:
"estimated cnn" is such a vague name that it could be detecting the moon phases. Please use a name that describes _what_ the thing is, not _how it is made_.